From 7c8690a06f49d17267e633e4ed6e9efc7f8d101d Mon Sep 17 00:00:00 2001
From: liyong4x <yongx.a.li@intel.com>
Date: Tue, 12 Jun 2018 20:34:27 -0400
Subject: [PATCH 1985/2367] media: intel-ipu4: ar0231at: Correct input clk and
 pclk

Message for Open Source:
The previous input clk and pclk value can work normally,
but not recommended. So change as the right values from
sensor vendor.

Message for Internal:
The previous input clk and pclk value can work normally,
but not recommended. So change as the right values from
sensor vendor.

[Issue/Feature] The previous input clk and pclk value are
not recommended by vendor.

[Root Cause/Changes]
1. change the input_clk as 27000000
2. change the op_sys_clk as 87750000

Change-Id: Ib3d2f8c302b2f0bdaf249be1fddad4e247350369
Tracked-On: #JIIAP-475
Signed-off-by: liyong4x <yongx.a.li@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h b/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
index 9ec2981..42b834f 100644
--- a/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
@@ -19,8 +19,8 @@
 
 struct crl_pll_configuration ar0231at_pll_configurations[] = {
 	{
-		.input_clk = 24000000,
-		.op_sys_clk = 264000000,
+		.input_clk = 27000000,
+		.op_sys_clk = 87750000,
 		.bitsperpixel = 12,
 		.pixel_rate_csi = 176000000,
 		.pixel_rate_pa = 176000000, /* pixel_rate = op_sys_clk*2 *csi_lanes/bitsperpixel */
-- 
2.7.4

